Currency.ts

```diff
@@ -553,6 +375,32 @@ export namespace Currency {
 		const factor = Math.pow(10, decimalDigits(currency) ?? 2)
 		return Math.round((value + Number.EPSILON) * factor) / factor
 	}
+	export function add(currency: Currency, value1: number, value2: number): number {
+		const decimals = decimalDigits(currency)
+		const factor = Math.pow(10, decimals == 0 ? 2 : decimals ?? 2)
+		const result = Math.round(factor * value1 + factor * value2) / factor
+		return decimals == 0 ? Math.round(result) : result
+	}
+	export function divide(currency: Currency, nominator: number, denominator: number): number {
+		const decimals = decimalDigits(currency)
+		const factor = Math.pow(10, decimals == 0 ? 2 : decimals ?? 2)
+		const result = Math.round(factor * (nominator / denominator)) / factor
+		return decimals == 0 ? Math.round(result) : result
+	}
+	export function multiply(currency: Currency, value1: number, value2: number): number {
```

Comment:
Just realized that you are multiplying a currency with a qouta (andel). Not a currency with a currency e.g. `SEK*SEK = SEK^2` makes no sense. So only value1 should be made into minor units. Same for division.

Also you should be able to calculate e.g. SEK to ISK. So I suggest a 4th optional `outputCurrency` argument.